testsuites/store/src/main/java/org/eclipse/rdf4j/sail/RDFStoreTest.java

```diff
@@ -718,6 +726,7 @@ public void testQueryBindings()
 	}
 
 	@Test
+	@Ignore("test is fundamentelly flawed as it uses multithreaded access on the same connection")
```

Comment:
I would change the focus to open N new connections for the N threads being tested and create join points using CountDownLatch's between the stages to verify the overall sequence is somewhat replicable and not completely random (no point having a test that isn't replicable to some degree, even if it is trying to test concurrency).